Resources/Prototypes/Loadouts/Miscellaneous/glasses.yml

```diff
@@ -0,0 +1,66 @@
+# Timers
+- type: loadoutEffectGroup
+  id: JamjarTimer
+  effects:
+  - !type:JobRequirementLoadoutEffect
+    requirement:
+      !type:DepartmentTimeRequirement
+      department: Science
+      time: 36000 #10 hours of being a NERD
+
+- type: loadoutEffectGroup
+  id: JensenTimer
+  effects:
+  - !type:JobRequirementLoadoutEffect
+    requirement:
+      !type:DepartmentTimeRequirement
+      department: Cargo
+      time: 36000 #10 hours of being a space trucker
+
+# Basic options
+# Glasses
+- type: loadout
+  id: Glasses
+  equipment: Glasses
+
+- type: startingGear
+  id: Glasses
+  equipment:
+    eyes: ClothingEyesGlasses
+
+# Cheap Sunglasses
+- type: loadout
+  id: CheapSunglasses
+  equipment: CheapSunglasses
+
+- type: startingGear
+  id: CheapSunglasses
+  equipment:
+    eyes: ClothingEyesGlassesCheapSunglasses
```

Comment:
i think it might be a balance issue for things like revs or security in general if there are a ton of fake sunglasses going around. It's hard to identify the difference at a glance.

I really would not include these in the loadout and just stick with regular glasses.